Makefile

```diff
@@ -167,11 +167,19 @@ dupl:
 .PHONY: check
 check:
 	@echo "checking for tabs in shell scripts"
-	@! git grep -F '	' -- '*.sh'
+	@! env PAGER='' git grep -F '	' -- '*.sh'
```

Comment:
We should probably put `export GIT_PAGER :=` at the top of the `Makefile` so that all `git` invocations will get this setting.